table/block_based_table_reader.cc

```diff
@@ -697,6 +697,8 @@ SequenceNumber GetGlobalSequenceNumber(const TableProperties& table_properties,
   }
 
   SequenceNumber global_seqno = DecodeFixed64(seqno_pos->second.c_str());
```

Comment:
I'm surprised that the existing code has no verification that seqno_pos is valid.

Reply:
Done.